zhaquirks/philips/hue_light.py

```diff
@@ -4,6 +4,14 @@
 
 from zhaquirks.philips import PHILIPS, SIGNIFY, PhilipsHueLightCluster
 
+(
+    QuirkBuilder()
+    .applies_to(SIGNIFY, "929004610402")
+    .friendly_name(model="Hue Flux Gradient lightstrip", manufacturer="Philips")
```

Comment:
We've apparently lower-cased "gradient" for the name for others, so we'll also do that ffor this model, for now.
We can adjust in the future. I'll also split this over two lines.
```suggestion
    .friendly_name(model="Hue Flux gradient lightstrip", manufacturer="Philips",)
```
And generally, it's better to add new devices at the bottom.